services/claim-backend/src/modules/claim/claim.service.ts

```diff
@@ -331,16 +336,20 @@ export class ClaimService {
       if (claim) {
         const nonce = claim.id * 13;
 
-        const claimContractInstance =
-          this.blockchainService.getContractInstance(
-            ContractTypes.CLAIM,
-            this.ethereumConfig.claimContractAddress,
-          );
-        const confirmation = await claimContractInstance.getClaimedAmount(
+        const indexedClaims = await this.graphService.getIndexedClaim(
           address,
           claim.numberOfTokens,
           nonce,
         );
+        if (!indexedClaims)
+          throw new InternalServerErrorException(
+            `No such claim was indexed with nonce: ${nonce}`,
+          );
+        if (indexedClaims.length > 1)
+          throw new InternalServerErrorException(
+            `Multiple claims were indexed with nonce: ${nonce}`,
+          );
+        const confirmation = ethers.BigNumber.from(indexedClaims[0].amount);
```

Comment:
We shouldn't throw here. This is the account request.

Reply:
```suggestion
```